target/linux/ramips/image/mt7621.mk

```diff
@@ -2898,6 +2898,17 @@ define Device/winstars_ws-wn583a6
 endef
 TARGET_DEVICES += winstars_ws-wn583a6
 
+define Device/wodesys_wd-r1802u
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
```

Comment:
I guess `$(Device/uimage-lzma-loader)` is useless on these new devices. It's a workaround for the old u-boot decompression issue.

Reply:
Original firmware used FIT format; I couldn't make it work with OpenWrt. Legacy uimage would not boot without the loader.

Reply:
I did try a FIT image at first and it wouldn't boot; that's why I went for the legacy format.
And I'm somewhat hesitant to try this again since it was a PITA to open up the device, and UART is needed to recover if it doesn't boot correctly on the first attempt.

Do you think I should attempt make it work with the FIT format? is there a tangible benefit to it (besides a quicker boot without the loader)?

Reply:
FIT image has some improvements for the security and multi dtbs support. But to be honest, I don't see any improvement for the ramips target. If you don't want to try again, just keep it.